webservices/schemas.py

```diff
@@ -252,8 +252,10 @@ class CommitteeSearchListSchema(ApiSchema):
 ScheduleEByCandidateSchema = make_schema(
     models.ScheduleEByCandidate,
     fields={
-        'committee': ma.fields.Nested(schemas['CommitteeHistorySchema']),
-        'candidate': ma.fields.Nested(schemas['CandidateHistorySchema']),
+        'committee_id': ma.fields.Str(),
```

Comment:
Sorry for the collateral refactor! I noticed that this query run significantly faster when we didn't nest the entire candidate and committee, and now that I've thought about it more, I don't see the benefit of nesting anyhow.